returnn/datasets/postprocessing.py

```diff
@@ -0,0 +1,195 @@
+__all__ = ["PostprocessingDataset"]
+
+from dataclasses import dataclass
+from numpy import ndarray
+from typing import Any, Callable, Dict, Iterator, List, Optional, Tuple
+
+from returnn.tensor import TensorDict
+from returnn.util.basic import NumbersDict
+from .basic import Dataset, init_dataset
+from .util.strings import str_to_numpy_array
+
+
+@dataclass
+class LoadedData:
+    tensors: TensorDict
+    seq_idx: int
```

Comment:
This should be moved to the end of the file. The public class should come first. Also, prefix it with `_` to make clear this is only used internally in the dataset.